language-server/modules/langserver-core/src/test/resources/completion/variable-declaration/config/project_var_def_ctx_config2.json

```diff
@@ -798,6 +798,30 @@
           "newText": "import ballerina/lang.regexp;\n"
         }
       ]
+    },
+    {
+      "label": "lsproject.models",
+      "kind": "Module",
+      "detail": "Module",
+      "sortText": "R",
+      "filterText": "models",
+      "insertText": "models1",
+      "insertTextFormat": "Snippet",
+      "additionalTextEdits": [
+        {
+          "range": {
+            "start": {
+              "line": 0,
+              "character": 0
+            },
+            "end": {
+              "line": 0,
+              "character": 0
+            }
+          },
+          "newText": "import lsproject.models as models1;\n"
```

Comment:
This is because in the LSPackageLoader we didn't check for already imported modules. So even if there's an imported module we suggest to reimport the same module.

Reply:
fixed with baa698f